Cargo.toml

```diff
@@ -28,7 +28,7 @@ dirs = "5.0.1"
 # Pinned to avoid including multiple versions of a dependency
 is-terminal = "=0.4.7"
 # Pinned to avoid downloading and running binary blobs on Linux
```

Comment:
Please remove this comment smile, otherwise LGTM

Reply:
Done, I also moved it to be next to the other `serde` dependencies.